plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -172,14 +188,15 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
         span.end();
       }
     };
+    return patchedFunctionWithOriginalProperties(patchedFunction, original);
   }
 
   protected _patchCallbackFunction<T extends (...args: any[]) => ReturnType<T>>(
     functionName: FMember,
     original: T
   ): T {
     const instrumentation = this;
-    return <any>function (this: any, ...args: any[]) {
+    const patchedFunction = <any>function (this: any, ...args: any[]) {
       const activeContext = api.context.active();
 
       if (!instrumentation._shouldTrace(activeContext)) {
```

Comment:
this was where the merge conflict were (same thing in above and below functions), I hope it's fine